index.html

```diff
@@ -13567,10 +13571,10 @@ <h3>States and Properties</h3>
 			<p>In general, [=user agents=] do not do much validation of <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> [=ARIA/properties=]. User agents MAY do some minor validation on request and enforce things like <pref>aria-posinset</pref> being within 1 and <pref>aria-setsize</pref>, inclusive. User agents are not  responsible for logical validation, such as the following:</p>
 				<ol>
 					<li>Circular references created by relations, such as specifying that two [=elements=] own each other. </li>
-					<li>Correct usage with regard to <abbr title="Document Object Model">DOM</abbr> tree structure, such as an <a>element</a> being owned by more than one other element.</li>
+					<li>Correct usage with regard to <abbr title="Document Object Model">DOM</abbr> tree structure.</li>
```

Comment:
I just didn't understand the point of the second half of this sentence so I removed it.

Reply:
My reading of this was that it's saying user agents are not responsible for validating something like this:

```html
<div aria-owns="child">content</div>
<div aria-owns="child">content 2<div>
<div id="child">I have two parents</div>
```

Reply:
ohhh I see ok got it. Well, I will put it back, also, it seems like "owns" make sense here since it actually means the reference of `aria-owns`